### DISCUSSION TITLE
配置里输出信息错误是怎么回事

### DISCUSSION THREAD

**Discussion by Asuna380** [mikumifa/biliTickerBuy#264] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/154870429/bff7ca98-5fae-4050-98b0-e42e855e25f1)

**Comment by JieKBK** [comment]
不要关掉终端窗口

**Reply by Asuna380** [reply]
没关掉啊

**Reply by peigenr** [reply]
那就重启软件 刷新网页

**Reply by Asuna380** [reply]
知道咋回事了，地址填错了，不能直接点浏览器默认的信息，要选才行
